packages/shared/src/graphql/posts.ts

```diff
@@ -38,6 +38,7 @@ export const supportedTypesForPrivateSources = [
   PostType.Share,
   PostType.Welcome,
   PostType.Freeform,
+  PostType.VideoYouTube,
```

Comment:
Wondering if we should simply extend `baseFeedSupportedTypes` here?
And include welcome?

Reply:
That makes sense, so it is going to be a lot easier.